server/src-lib/Hasura/Server/Logging.hs

```diff
@@ -221,14 +223,20 @@ mkHttpErrorLog userInfoM reqId req err query mTimeT compressTypeM =
            , olUserVars     = userVars <$> userInfoM
            , olResponseSize = respSize
            , olQueryExecutionTime = respTime
-           , olQuery = toJSON <$> query
+           , olQuery = toJSON <$> query'
+           , olRawQuery = toJSON <$> rawQuery
            , olError = Just $ toJSON err
```

Comment:
There should be no need to call `toJSON` explicitly here if the types are updated as mentioned above, since the values should already have the appropriate types.